pom.xml

```diff
@@ -185,7 +185,10 @@
 		<module>xchange-wex</module>
 		<module>xchange-yobit</module>
 		<module>xchange-zaif</module>
-	</modules>
+        <module>xchange-coindeal</module>
+        <!--<module>xchange-coinex</module>-->
```

Comment:
you really want this commented out?

Reply:
Coinex isn't completed like the rest exchanges, that is why i have comment it. I am adding it in order more people can update it but i don't want to included it on the project yet. Maybe i am wrong here, feel fee to tell me your opinion,about this.Thanks